tools/slicec-cs/src/generators/exception_generator.rs

```diff
@@ -49,23 +49,21 @@ pub fn generate_exception(exception_def: &Exception) -> CodeBlock {
     );
 
     if !exception_def.all_fields().is_empty() {
-        exception_class_builder.add_block(one_shot_constructor(exception_def));
-
-        // Also generate a parameterless constructor for decoding.
+        // Parameterless constructor.
         exception_class_builder.add_block(
             FunctionBuilder::new("public", "", &exception_name, FunctionType::BlockBody)
-                .add_never_editor_browsable_attribute()
                 .add_comment(
                     "summary",
-                    format!(
-                        r#"Constructs a new instance of <see cref="{}"/> for the Slice decoder."#,
-                        &exception_name
-                    ),
+                    format!(r#"Constructs a new instance of <see cref="{}"/>."#, &exception_name),
                 )
-                .set_body(initialize_required_fields(fields))
                 .build(),
         );
+
+        exception_class_builder.add_block(one_shot_constructor(exception_def));
+
+        // TODO: generate secondary constructor like for classes.
```

Comment:
It's unfortunate that class_generator and exception_generator are so different, when the logic and generated code are almost identical. Fixing this inconsistency is outside the scope of this PR.